tests/functional_tests.rs

```diff
@@ -48,9 +40,7 @@ async fn main() {
         username.as_str()
     );
 
-    Distributions::from("ubuntu")
-        .create_user(username.as_str(), "")
-        .expect("Failed to create user");
+    Distributions::from("ubuntu").create_user(username.as_str(), "")?;
```

Comment:
Is there a way to keep an error context of error message `Failed to create user`?
Simply dropping the message looks a little backwards.

If `anyhow` is imported in the code base, we could have `.context()`, but that is a long term goal for now.

Reply:
Yes, let's split functional tests into a separate PR.